docs/source/querying/querybuilder/append.rst

```diff
@@ -127,7 +127,7 @@ This will be the next example::
     qb.append(
         CalcJobNode,
         filters={
-            'state':{'in':['FINISHED', 'RETRIEVING']}
+            'attributes.process_state':{'in':['finished', 'retrieving']}
```

Comment:
`retrieving` is not a valid `process_state`. Please replace it by `excepted` and or `killed`, which are valid `ProcessState` values.

Reply:
Thanks. I pushed this fix to my clone's develop. I don't know the correct manner of PR. Was this OK?

Reply:
This was fine. For your own convenience, typically it is nicer to make a specific branch for a fix. You can do that by doing `git checkout -b fix_some_isse` and make your commits there. Then just push that branch to your fork and you can create the PR from there. Then there is one final thing: as you can see now, your branch is "out-of-data", meaning our develop has commits that your branch does not have yet. Typically when you push your PR branch, make sure it is up to date by checking out your branch and doing `git fetch origin && git rebase origin/develop`. This will update your branch to include the latest changed of our develop

Reply:
OK. I do it the next. Thanks for your explanation!